Issue 1

belodpav opened the pull request:
Еще как вариант, перенести все линтеры клиента(стайллинт) в зависимости клиента и конфиги туда же. Что думаете?

belodpav committed:
(new, issue-1) Создаст init структуру проекта

belodpav committed:
(issue-1) Создаст главный файл сервера

belodpav committed:
(docs) Создаст init README файл

belodpav committed:
(docs) Создаст init README для client

belodpav committed:
(del) Удалит неиспользованный файл-заглушку

belodpav committed:
(issue-1) Удаляет мусор из index.html

belodpav committed:
(issue-1) Обновит manifest.json

Для изменения имени приложения по умолчанию

belodpav committed:
(del) Удалит лишние файлы "по умолчанию"

belodpav committed:
(issue-1) Установит правила для eslint

belodpav committed:
(config) Создаст init настройки для линтера

belodpav committed:
(config) Обновит package.json для линтера

belodpav committed:
(issue-1) Добавит EOF

belodpav committed:
(config) Настроит eslint для client

belodpav committed:
(config) Обновит eslint config с учетом среды

belodpav committed:
(config) Установит зависимости для `stylelint

belodpav committed:
(issue-1) Добавит .css файл для тестирования

belodpav committed:
(config) Настроит конфиг для stylelint

Использутся стандартная конфигурация

belodpav committed:
(config) Настраивает установку пакетов приложения

belodpav committed:
(issue-1) Добавляет EOF в файл кофига

belodpav committed:
(docs) Добавит интрукцию по установке

JorJeG commented:
в корне репозитория нам не нужен .stylelint

belodpav commented:
@JorJeG  Ок. Изменим. 
Жор, назначь ревьюверов еще.

JorJeG commented:
@belodpav Я сейчас посмотрел у себя, если переместить просто файл .stylelintrc.json в client, то всё будет работать в VSCode.
даже можно удалить .stylelintignore потому, что он внешнии директории не увидит

belodpav committed:
(issue-1) Переместит stylelint конфиги в client

belodpav commented:
@JorJeG 

1.  .stylelintignore надо сохранить. И в преспективе улучшить. Он позволяет игнорировать node_modules и т.п. Что сильно влияет на время проверки.
2. Важнее чтобы проходила cli команда npm run lint, чем подсветка в vscode

JorJeG committed:
(ref) Переместит stylelint только на client

Также поправит stylelint script, потому что теперь нужно ссылаться
на stylelint, который установлен на уровне client

JorJeG committed:
(config) Добавит config для vscode

Позволит использовать плагин stylelint в vscode для подсветки ошибок

dezmound reviewed (commented):
Я думаю стоит включить package-lock.json в .gitignore и удалить его из репозитория.

belodpav committed:
(issue-1) Удаляет избыточные настройки IDE

belodpav committed:
(issue-1) Обновит package-lock.json

belodpav commented:
Больших проблем больше не вижу. Если что в процессе подправим. Чтобы не стоять на месте рекоммендую замержить

belodpav committed:
(config) Обновит gitignore с учетом папки static

belodpav committed:
(config) Добавляет скрипт переноса статики

При сборке production версии клиентского
приложения полученная статика переносится
в окружение сервера это позволяет сделать
более логично-обоснованную архитектуру
клиент-серверного приложения
